mtools/mplotqueries/mplotqueries.py

```diff
@@ -198,6 +201,9 @@ def parse_logevents(self):
                     self.progress_bar_enabled = False
 
             for i, logevent in enumerate(logfile):
+                if (self.args['dns'] and (
```

Comment:
is the key 'dns' always part of the args passed to the tool? what if is not there? is there a default value?

Reply:
--dns acts as a filter to parse (and plot) only lines containing DNS log entries. if --dns argument is not provided then all log lines will be parsed and plotted. by default, the parser plots all lines containing an operation or command.